.github/workflows/tests.yml

```diff
@@ -3,22 +3,27 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8.3', '3.7.8', '3.6.11']
+    name: Python ${{ matrix.python-version }}
     steps:
-      - uses: actions/checkout@v1
-      - name: Install pyenv
-        uses: gabrielfalcao/pyenv-action@v5
-        with:
-          default: 3.8.2
-          versions: 3.8.2 3.7.7 3.6.10
-      - name: Install dependencies
-        uses: abatilo/actions-poetry@v1.5.0
-        with:
-          python_version: 3.8
-          poetry_version: 1.0.5
-          args: install
-      - name: Run tests
-        uses: abatilo/actions-poetry@v1.5.0
-        with:
-          python_version: 3.8
-          poetry_version: 1.0.5
-          args: run tox
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Poetry
+      run: pip install poetry
+    - name: Install dependencies
+      run: poetry install
+    - name: Black
+      run: poetry run black .
+    - name: Tox 3.8
+      if: ${{ matrix.python-version  == '3.8.3' }}
+      run: poetry run tox -e 'py38-django{2,3}'
```

Comment:
I think you can eliminate the repetition by using only one step with `with` and `python-version`. See an example [here](https://github.com/model-bakers/model_bakery/pull/89/files).

Reply:
I will try! Let's see.

Reply:
@anapaulagomes Can you check now?  I use [tox-gh-actions](https://github.com/ymyzk/tox-gh-actions) and configured env vars on `tox.ini` file. Thank you for the tip!